src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/util/ExceptionRecordAttachDocumentConnectives.java

```diff
@@ -9,14 +9,25 @@
      */
     private Set<String> duplicates;
 
+    /**
+     * Exception Record documents which are still missing in target case.
+     */
+    private Set<String> missing;
+
     public ExceptionRecordAttachDocumentConnectives(
-        Set<String> duplicates
+        Set<String> duplicates,
+        Set<String> missing
     ) {
         this.duplicates = duplicates;
+        this.missing = missing;
     }
 
     public boolean hasDuplicates() {
-        return !duplicates.isEmpty();
+        return !duplicates.isEmpty() && !missing.isEmpty();
```

Comment:
Not sure my understanding of implied logic is correct but the logical expression does not look matching the name of the method